tests/pages/api/grafanaAPI.js

```diff
@@ -166,15 +166,22 @@ module.exports = {
   async getMetric(metricName, refineBy) {
     const timeStamp = Date.now();
     const bodyFormData = new FormData();
-    const body = {
+    let body;
+
+    body = {
       query: metricName,
       start: Math.floor((timeStamp - 15000) / 1000),
       end: Math.floor((timeStamp) / 1000),
-      step: 60,
+      step: 1,
     };
 
     if (refineBy) {
-      body.query = `${metricName}{${refineBy.type}=~"(${refineBy.value})"}`;
+      body = {
+        query: `${metricName}{${refineBy.type}=~"(${refineBy.value})"}`,
+        start: Math.floor((timeStamp - 10000) / 1000),
```

Comment:
in line 173 value is 15 000 and here is 10 000 is this intentionally?
also is there a benefit to reassign complete body instead of change only 2 fields?

Reply:
Good questions, no it doesn't make any difference and your point is right, I was trying to make it work with original, reverted it